--- work.orig/setup.py	2018-08-30 13:04:29.106053556 -0400
+++ work/setup.py	2018-08-30 13:04:58.076766218 -0400
@@ -369,10 +369,7 @@
     from setuptools import setup
     if run_build:
         from numpy.distutils.core import setup
-        cwd = os.path.abspath(os.path.dirname(__file__))
-        if not os.path.exists(os.path.join(cwd, 'PKG-INFO')):
-            # Generate Cython sources, unless building from source release
-            generate_cython()
+        generate_cython()
 
         metadata['configuration'] = configuration
     else:
