diff -urN a/numpy/core/tests/test_umath.py b/numpy/core/tests/test_umath.py
--- a/numpy/core/tests/test_umath.py	2019-12-27 17:24:44.000000000 +0300
+++ b/numpy/core/tests/test_umath.py	2022-06-10 11:08:57.409097639 +0300
@@ -2419,6 +2419,7 @@
         b = 5*big
         assert_equal(np.lcm(a, b), 10*big)
 
+    @pytest.mark.skip(reason="Reported here but never resolved? -> https://github.com/numpy/numpy/issues/16046")
     def test_gcd_overflow(self):
         for dtype in (np.int32, np.int64):
             # verify that we don't overflow when taking abs(x)
